src/applications/vaos/components/tests/VideoLayoutAtlas.unit.spec.js

```diff
@@ -33,6 +33,79 @@ describe('VAOS Component: VideoLayoutAtlas', () => {
     },
   };
 
+  describe('When appointment information is missing', () => {
+    it('should not display heading and text for empty data', async () => {
+      // Arrange
+      const store = createTestStore(initialState);
+      const appointment = {
+        comment: 'This is a test:Additional information',
+        location: {
+          stationId: '983',
+          clinicName: 'Clinic 1',
+          clinicPhysicalLocation: 'CHEYENNE',
+        },
+        videoData: {
+          atlasConfirmationCode: '1234',
+          atlasLocation: {
+            address: {
+              line: ['5929 Georgia Ave NW'],
+              city: 'Washington',
+              state: 'DC',
+              postalCode: '20011',
+            },
+          },
+          isVideo: true,
+          facilityId: '983',
+          isAtlas: true,
+          kind: VIDEO_TYPES.adhoc,
+          extension: {
+            patientHasMobileGfe: false,
+          },
+        },
+        vaos: {
+          isCommunityCare: false,
+          isCompAndPenAppointment: false,
+          isCOVIDVaccine: false,
+          isPendingAppointment: false,
+          isUpcomingAppointment: true,
+          isVideo: true,
+          apiData: {},
+        },
+        status: 'booked',
+      };
+
+      // Act
+      const screen = renderWithStoreAndRouter(
+        <VideoLayoutAtlas data={appointment} />,
+        {
+          store,
+        },
+      );
+
+      // Assert
+      expect(
+        screen.getByRole('heading', {
+          level: 1,
+          name: /Video appointment at an Atlas location/i,
+        }),
+      );
+
+      expect(
+        screen.queryByRole('heading', {
+          level: 2,
+          name: /Who/i,
+        }),
+      ).not.to.exist;
+
+      expect(
+        screen.queryByRole('heading', {
+          level: 2,
+          name: /What/i,
+        }),
+      ).not.to.exist;
```

Comment:
Add assertions for missing facility and clinic info.

Reply:
@JunTaoLuo I've added these assertions